### PR TITLE
Created stepper component, also added some styles to it

### PR DESCRIPTION
**Pull request by SerdarAtamuradov**

### CODE REVIEW

**Comment by RomPavel** [comment]
Please, add screenshot when you make UI changes

**Comment by SerdarAtamuradov** [comment]
Снимок экрана (1177)

**Comment by RomPavel** [comment]
Could you please make it look like more like a test.
Аddress mocks are super unrelated to our project )

File: exadel-trainee-app/src/app/app.module.ts [hunk 1]
```diff
@@ -21,7 +22,7 @@ import { StatisticsComponent } from './components/mock-component/statistics/stat
     ManageComponent,
     StatisticsComponent,
   ],
-  imports: [BrowserModule, AppRoutingModule, BrowserAnimationsModule, SidebarModule],
+  imports: [BrowserModule, AppRoutingModule, BrowserAnimationsModule, SidebarModule, StepperModule],
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/app.module.ts:25]: Its better to not import this module into main module
We need it only for listening module and grammar module if Iam not mistaken

File: exadel-trainee-app/src/app/components/stepper/stepper.component.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+.stepper {
+  width: 80%;
+  margin: 0 auto;
+}
+
+.button__color {
+  color: #c7c7c7;
+  background-color: #555555;
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/components/stepper/stepper.component.scss:12]: We have **$secondary-grey2** scss constant for this color

File: exadel-trainee-app/src/app/components/stepper/stepper.component.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+.form__color {
+  color: #c7c7c7;
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/components/stepper/stepper.component.scss:16]: Maybe you can use **$secondary-grey3** const here

File: exadel-trainee-app/src/app/components/stepper/stepper.module.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+@NgModule({
+  declarations: [StepperComponent],
+  imports: [
+    CommonModule,
+    MatFormFieldModule,
+    MatButtonModule,
+    MatStepperModule,
+    FormsModule,
+    ReactiveFormsModule,
+    MatInputModule,
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/components/stepper/stepper.module.ts:21]: Now we have **AngularMaterialCommonModule** for importing/exporting of angular/material modules

**Comment by SerdarAtamuradov** [comment]
image

File: exadel-trainee-app/src/app/components/stepper/stepper.module.ts (hunk 4 above)
Comment (RomPavel) [reply exadel-trainee-app/src/app/components/stepper/stepper.module.ts:19]: Now you need to import **AngularMaterialCommonModule** into **StepperModule** and remove duplication modules

File: exadel-trainee-app/src/app/models/listening-test/listening-test.component.html [hunk 5]
```diff
@@ -1 +1,2 @@
-<p>listening works!</p>
+<!-- <p>listening works!</p> -->
+<app-stepper></app-stepper>
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/models/listening-test/listening-test.component.html:2]: Could you please add it to grammar component and grammar module

File: exadel-trainee-app/src/app/models/angular-material-common.module.ts [hunk 6]
```diff
@@ -26,6 +31,9 @@ import { MatButtonModule } from '@angular/material/button';
     MatInputModule,
     MatSortModule,
     MatButtonModule,
+    MatStepperModule,
+    FormsModule,
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/models/angular-material-common.module.ts:35]: Lets not add **FormsModule** and **ReactiveFormsModule**
This module named **AngularMaterialCommonModule** so everybody expect that it only contains modules from angulat/material

**Comment by SerdarAtamuradov** [comment]
image

Added stepper into grammar module

**Comment by SerdarAtamuradov** [comment]
image

Changes with color

**Review by RomPavel — APPROVED** [review]

**Pull request merged**